workspaces/theme/plugins/theme/src/utils/createComponents.ts

```diff
@@ -424,14 +424,26 @@ export const createComponents = (themeConfig: ThemeConfig): Components => {
           padding: '0 1.5rem',
         },
         vertical: {
+          '& > div > div > button[class*="v5"]': {
+            alignItems: 'baseline',
+            textAlign: 'left',
+          },
+          '& > div > div > button > span': {
+            alignItems: 'baseline',
+            textAlign: 'left',
+          },
```

Comment:
A small thing we could improve later. You can also use a class selector like this:

```suggestion
          // MUI v5
          '& > div > div > button.v5-MuiTab-root': {
            alignItems: 'baseline',
            textAlign: 'left',
          },
          // MUI v4
          '& > div > div > button > span': {
            alignItems: 'baseline',
            textAlign: 'left',
          },
```